README.md

```diff
@@ -79,17 +79,7 @@ $ cd pbench
 
 ### Running the Unit Tests
 
-To simply run the unit tests quickly from within the checked out source tree,
-execute:
-
- * `jenkins/run jenkins/tox -r --current-env -e jenkins-pytests`
- * `jenkins/run jenkins/tox -r --current-env -e jenkins-unittests`
-
-The above commands run the tests in a Fedora-base container with all the
-proper packages installed.
```

Comment:
I think it's really important for developers to be able to run the tests exactly like the CI will run them, and I think it would be appropriate for the CI to run tests using the same mechanism that the developers will use.  So, I think simply deleting this text (and removing `jenkins/tox`) is not moving things in the right direction.

Reply:
See https://github.com/distributed-system-analysis/pbench/blob/29f2e04832e075c967d56bcf84cbe0e4322d923c/jenkins/Pipeline.gy#L14 -- the CI and developer mode of running tests are very different.  This is yet a 3rd way to run them.

Reply:
Your statement that they "are" very different makes me regret my failure to delete all traces of Tox when I "fixed" the CI -- had I done so, then there would be no argument here....

And, given that my previous PR left us in a transitional state which was never completed, I would prefer that this PR not produce the analogous result.

Reply:
We are going back the other way.  From what I can tell, `tox` gives us the best way for us to run tests in multiple python environments.